eng/scripts/Yank-Crates.ps1

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env pwsh
+
+#Requires -Version 7.0
+param(
+  [string]$PackagesPath,
+  [string[]]$CrateNames,
+  [string]$Token
+)
+
+$ErrorActionPreference = 'Stop'
+#Set-StrictMode -Version 2.0
+
+$hasErrors = $false
+foreach ($crateName in $crateNames) {
+  $crate = Get-Content "$PackagesPath/$crateName/cargo-metadata.json" -Raw | ConvertFrom-Json
+  $crateVersion = $crate.vers
+
+  Write-Host "Yanking crate: '$crateName@$crateVersion'"
+
+  # https://doc.rust-lang.org/cargo/reference/registry-web-api.html#yank
+  $response = Invoke-WebRequest -Method Delete -Uri "https://crates.io/api/v1/crates/$crateName/$crateVersion/yank" `
```

Comment:
Interesting: I'm surprised they take a `DELETE` to a ".../yank" URI instead of the resource (version) itself. That's just...not very RESTful. /cc @JeffreyRichter

Reply:
They're very not restful